### ISSUE TITLE
Add Support for Google Downloadable Fonts?

### ISSUE THREAD

**Issue by ashiqursuperfly** [InflationX/Calligraphy#57]
Been using calligraphy for a while now. However, theres an issue i would like to share. The only way to use calligraphy is via including your own .ttf files. (afaik ;_; correct me if im wrong) **Since, .ttf files can get large for most fonts, they contribute heavily to the overall app size**. Therefore, **Google downloadable fonts** are a much light weight alternative for custom fonts in android now.

It would be great if we could **set a google downloadable font as the calligraphy global font.** I dont know if this is possible in the current implementation (if at all) but will really appreciate this feature.

**Comment by willis-tagheuer** [comment]
I would love to see a similar feature to be able to download ttf at runtime

**Comment by chrisjenx** [comment]
Probably unlikely as that would mean we would have to include something like OkHttp in the project and we try to keep things lightweight.

**Comment by Pitel** [comment]
What might be enough is to provide a way to set the `Typeface` object for the font, and not just the static path. Because the result of getting the downloadable font is the `Typeface`.

**Comment by chrisjenx** [comment]
True, I don't really have time look right now. But between ViewPump and Calligraphy should be possible, happy to look and shape up PR's if you have time to submit one
